src/msw/rt/notifmsgrt.cpp

```diff
@@ -418,13 +418,31 @@ wxString wxToastNotifMsgImpl::ms_appId;
 wxCOMPtr<IToastNotificationManagerStatics> wxToastNotifMsgImpl::ms_toastMgr;
 
 HRESULT wxToastEventHandler::Invoke(
-    IToastNotification *WXUNUSED(sender),
-    IInspectable *WXUNUSED(args))
+    IToastNotification *sender,
+    IInspectable *args)
 {
     if ( m_impl )
     {
-        wxCommandEvent evt(wxEVT_NOTIFICATION_MESSAGE_CLICK);
-        m_impl->ProcessNotificationEvent(evt);
+        IToastDismissedEventArgs *dea = nullptr;
+        IToastFailedEventArgs *fea = nullptr;
+
+        if ( SUCCEEDED(args->QueryInterface(__uuidof(IToastDismissedEventArgs), (void**)&dea)) )
```

Comment:
We need to call `Release()` on `dea` in case of success to avoid memory leaks (and the same for `fea` below, of course).